src/hooks/useStatsig.tsx

```diff
@@ -0,0 +1,21 @@
+import { StatsigClient } from '@statsig/js-client';
+import { StatsigProvider, useStatsigClient } from '@statsig/react-bindings';
+
+export enum StatSigFlags {
+  // When adding a flag here, make sure to add an analytics tracker in useAnalytics.ts
+  ffSkipMigration = 'ff_skip_migration',
+}
+
+export const StatSigProvider = ({ children }: { children: React.ReactNode }) => {
+  const client = new StatsigClient(`${import.meta.env.VITE_STATSIG_CLIENT_KEY}`, {
```

Comment:
should this stuff be in a useEffect or useMemo or something? seems like it's gonna call these functions a lot

Reply:
they might even be fine to be declared in the global scope?